src/components/ChooseCheckGroup/Protocol.tsx

```diff
@@ -14,30 +14,25 @@ export const Protocol = ({ href, label, tooltip }: ProtocolOption) => {
   if (tooltip) {
     return (
       <Toggletip content={<div>{tooltip}</div>}>
-        <button className={styles.badgeLink}>
-          <Badge
-            text={
-              <Stack gap={0.5} alignItems={`center`}>
-                <div>{label}</div>
-                <Icon name={`info-circle`} size="sm" />
-              </Stack>
-            }
-            color={BADGE_COLOR}
-          />
-        </button>
+        <div className={styles.badgeLink}>
+          <Stack gap={0.5} alignItems={`center`}>
+            <span>{label}</span>
+            <Icon name={`info-circle`} size="sm" />
+          </Stack>
+        </div>
       </Toggletip>
     );
   }
 
   if (href) {
     return (
-      <a className={styles.badgeLink} href={href}>
-        <Badge text={label} color={BADGE_COLOR} />
-      </a>
+      <TextLink className={styles.badgeLink} href={href} inline={false} color="secondary">
```

Comment:
The color threw me off when I was using a mobile device.
I Wasn't ready for the "protocols" to be clickable. And if I would expect "protocols" to be clickable, "WebSockets" isn't 🤷🏻 
![image](https://github.com/user-attachments/assets/07985b0d-c754-46ef-8b0d-881211a7af00)

vs `primary`
![image](https://github.com/user-attachments/assets/2f30b6e5-f495-4bd9-93e7-5f685c372866)